renkuaqs/plugin.py

```diff
@@ -494,6 +496,30 @@ def display(revision, paths, filename, no_oda_info, input_notebook):
     pydot_graph.write_png(filename)
 
 
+@aqs.command()
+def start_session():
+    gitlab_url = subprocess.check_output(["git", "remote", "get-url", "origin"]).decode().strip()
+
+    new_session_urls = []
+
+    for pattern in [
+        'https://renkulab.io/gitlab/(.*)\.git',
+        'git@renkulab.io:(.*)\.git'
+    ]:
```

Comment:
Can those be configurable?

Reply:
This has to do with launching in renkulab. We have only only renkulab. What is the purpose of configuring them?
If we had multiple renku deployments it would be useful, but we are not currently even planning to use any extra instances. It's essential since single instance is also the workflow catalog.

Reply:
Ok makes sense.